Example/Emission/ARRootViewController.m

```diff
@@ -84,8 +84,7 @@ - (ARCellData *)jumpToArtist
 
 - (ARCellData *)jumpToRandomArtist
 {
-  NSDictionary *metadata = [[NSBundle mainBundle] infoDictionary];
-  NSString *sourceRoot = [metadata objectForKey:@"SourceRoot"];
+  NSString *sourceRoot = [NSProcessInfo processInfo].environment[@"SRCROOT"];
```

Comment:
nvm this is probably explained in the post (which I opened in a new tab, yet to read :P)